whobpyt/optimization/cost_PSD.py

```diff
@@ -2,6 +2,10 @@
 
 
 class CostsPSD():
+    '''

```

Comment:
Suggestion: Add a line to warn for the deprecation. a la [this link](https://dev.to/hckjck/python-deprecation-2mof#:~:text=Module%20deprecation,top%20level%20of%20that%20module).

For instance, define this before the class is initiated.
```python
from warnings import warn

def a_deprecated_function():
    warn('This method is deprecated.', DeprecationWarning, stacklevel=2)
```
Then adding this under `init`:
```python
warn(f'{self.__class__.__name__} will be deprecated.', DeprecationWarning, stacklevel=2)
```

Reply:
This function should be updated in the future, but message is okay for now.